services/ext/api.go

```diff
@@ -294,13 +294,16 @@ func (api *PublicAPI) ActiveChats(parent context.Context) []*protocol.Chat {
 func (api *PublicAPI) DeleteChat(parent context.Context, chatID string) error {
 	return api.service.messenger.DeleteChat(chatID)
 }
+func (api *PublicAPI) MuteCommunityCategoryV2(request *requests.MuteCategory) error {
+	return api.service.messenger.SetMutePropertyOnChatsByCategory(request, true)
+}
 
 func (api *PublicAPI) MuteCommunityCategory(communityID string, categoryID string) error {
-	return api.service.messenger.SetMutePropertyOnChatsByCategory(communityID, categoryID, true)
+	return api.service.messenger.SetMutePropertyOnChatsByCategory(&requests.MuteCategory{CommunityID: communityID, CategoryID: categoryID, MutedType: 0}, true)
```

Comment:
I'd use a constant here instead

Reply:
I did it in the same way as it was for `chats`, but now I realized why `0` when should be `5` (`MuteTillUnmuted`).